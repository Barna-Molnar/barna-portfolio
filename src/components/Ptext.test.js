import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import Ptext from './Ptext';

describe('Ptext test', () => {
  test('true === true', () => {
    expect(true).toBe(true);
  });

  test('renders children', () => {
    const renderedComp = render(<Ptext>Barni</Ptext>);

    expect(renderedComp.getByText('Barni')).toBeTruthy();
  });

  test('renders children', () => {
    const renderedComp = render(<Ptext>Almos</Ptext>);

    expect(renderedComp.getByText('Almos')).toBeTruthy();
  });

  test('renders children', () => {
    const renderedComp = render(
      <Ptext>
        <span>Almos</span>
      </Ptext>
    );

    expect(renderedComp.baseElement).toMatchSnapshot();
  });
});
