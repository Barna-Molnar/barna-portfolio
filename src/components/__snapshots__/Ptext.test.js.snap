// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ptext test renders children 1`] = `
<body>
  <div>
    <div
      class="sc-bdDrOH gijhgW para"
    >
      <p>
        <span>
          Almos
        </span>
      </p>
    </div>
  </div>
</body>
`;
